### PR TITLE
Adds thelio-mira-r2

### PR DESCRIPTION
**Pull request by XV-02**

### CODE REVIEW

**Review by leviport — DISMISSED** [review]
Makes mira-r2 show up in the Firmware section of GCC

**Review by 13r0ck — DISMISSED** [review]

**Comment by jackpot51** [comment]
This will need to be rebased

File: debian/changelog [hunk 1]
```diff
@@ -1,5 +1,9 @@
 system76-firmware (1.0.37) jammy; urgency=medium
 
+  * Add thelio-mira-r2
```
Comment (jackpot51) [inline debian/changelog:3]: This changelog format is not quite right, needs a new entry

**Comment by jackpot51** [comment]
Cleaned up, pushed to master

**Pull request closed without merging**